scripts/pre_commit_linter.py

```diff
@@ -984,7 +984,8 @@ def _check_comments(all_files):
                         print '%s --> Line %s: %s' % (
                             filename, line_num + 1, message)
 
-                # Check that comment starts with a space.
+                # Check that comment starts with a space
+                # and is not a shebang, which doesn't work with a space.
```

Comment:
This is still just repeating what a "#!" is. In general, though, when writing comments, try to explain the significance, or "why", of the decisions that are being made in the code.

For example, in this case, you can say something like: "Check that the comment starts with a space, unless it is part of the shebang expression at the start of a bash script." That explains to the reader why that particular case is excluded.

Reply:
Done I think, TAL